examples/OpenGL45/matrix_multiplication/matrix_multiplication_version2.shadertrap

```diff
@@ -0,0 +1,318 @@
+        6656. 6656. 6656. 6656. 7168. 7168. 7168. 7168. 7680. 7680. 7680. 7680. 8192. 8192. 8192. 8192.
+        512. 512. 512. 512. 1024. 1024. 1024. 1024. 1536. 1536. 1536. 1536. 2048. 2048. 2048. 2048. 
+        2560. 2560. 2560. 2560. 3072. 3072. 3072. 3072. 3584. 3584. 3584. 3584. 4096. 4096. 4096. 4096. 
+        4608. 4608. 4608. 4608. 5120. 5120. 5120. 5120. 5632. 5632. 5632. 5632. 6144. 6144. 6144. 6144. 
+        6656. 6656. 6656. 6656. 7168. 7168. 7168. 7168. 7680. 7680. 7680. 7680. 8192. 8192. 8192. 8192.
+        512. 512. 512. 512. 1024. 1024. 1024. 1024. 1536. 1536. 1536. 1536. 2048. 2048. 2048. 2048. 
+        2560. 2560. 2560. 2560. 3072. 3072. 3072. 3072. 3584. 3584. 3584. 3584. 4096. 4096. 4096. 4096. 
+        4608. 4608. 4608. 4608. 5120. 5120. 5120. 5120. 5632. 5632. 5632. 5632. 6144. 6144. 6144. 6144. 
+        6656. 6656. 6656. 6656. 7168. 7168. 7168. 7168. 7680. 7680. 7680. 7680. 8192. 8192. 8192. 8192.
+
+CREATE_BUFFER mat_mul_expected SIZE_BYTES 1024 INIT_VALUES 
+    float 512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+          512. 1024. 1536. 2048. 2560. 3072. 3584. 4096. 4608. 5120. 5632. 6144. 6656. 7168. 7680. 8192.
+
+DECLARE_SHADER array_to_blocked_array KIND COMPUTE
+#version 450
+#define KERNEL_SIZE 4
+#define MAT_SIZE  16
+
+
+layout(local_size_x=1, local_size_y=1, local_size_z=1) in;
+
+layout(std430, binding = 0) buffer array1 {
+  float[MAT_SIZE][MAT_SIZE] data;
```

Comment:
Change it so that the array sizes come after `data` - please do the same throughout the PR.

Reply:
@EGJ1996 looks like you missed this one.